MyClass.java

```diff
@@ -0,0 +1,15 @@
+package com.foo.bar;
+
+import org.kohsuke.github.authorization.AuthorizationProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.time.Instant;
+import javax.annotation.Nonnull;
```

Comment:
```suggestion

```

<details>
  <summary>More Info</summary>
  ~~~
  ### Help! This doesn't look right!
  Please let us know in #developer-platform

  ### How was this generated?
  This fix was generated from the following SARIF:
  ```json
  {
  "ruleId" : "java:S1128",
  "message" : {
    "text" : "Remove this unused import 'java.time.Duration'."
  },
  "locations" : [ {
    "physicalLocation" : {
      "artifactLocation" : {
        "uri" : "MyClass.java"
      },
      "region" : {
        "startLine" : 9,
        "startColumn" : 7,
        "endLine" : 9,
        "endColumn" : 25
      }
    }
  } ],
  "fixes" : [ {
    "description" : {
      "text" : "Remove the import"
    },
    "artifactChanges" : [ {
      "artifactLocation" : {
        "uriBaseId" : "MyClass.java"
      },
      "replacements" : [ {
        "deletedRegion" : {
          "startLine" : 9,
          "endLine" : 9
        },
        "insertedContent" : {
          "text" : ""
        }
      } ]
    } ]
  } ]
}
  ```

  Resulting in this fix:
  ```json
  {
  "text" : "",
  "startLine" : 9,
  "endLine" : 9
}
  ```
</details>